Dockerfile

```diff
@@ -23,13 +23,13 @@ WORKDIR ${folder}
 COPY . /home/app/mava
 # For box2d
 RUN apt-get install swig -y
-## Install core dependencies.
-RUN pip install -e .[reverb,launchpad]
+## Install core dependencies + reverb.
```

Comment:
Rather `install reverb dependencies`

Reply:
So I think this installs both core (`.` does core) and reverb.